charts/cloudzero-agent/configuration.example.yaml

```diff
@@ -12,7 +12,7 @@ global:
   existingSecretName: null
 
 # label and annotation configuration:
-tags:
+insightsController:
```

Comment:
I'm making this change because I want to make sure that we choose a name that will make sense for future changes. the insights-controller can potentially handle much more than labels and annotations, so I don't want to add a confusing name. 
the downside is that this is less descriptive for users who are looking to configure labels/annotations